pha-admin/src/main/java/com/store/pharmacy/bizpartner/service/impl/BizPartnerServiceImpl.java

```diff
@@ -0,0 +1,192 @@
+package com.store.pharmacy.bizpartner.service.impl;
+import com.store.pharmacy.bizpartner.model.BizPartner;
+import com.store.pharmacy.bizpartner.model.BizPartnerInput;
+import com.store.pharmacy.bizpartner.model.BizPartnerOutput;
+import com.store.pharmacy.bizpartner.repository.BizPartnerRepository;
+import com.store.pharmacy.bizpartner.service.BizPartnerService;
+import com.store.pharmacy.common.model.Common;
+import com.store.pharmacy.common.repository.CommonRepository;
+import com.store.pharmacy.common.service.LabelService;
+import com.store.pharmacy.exception.DataNotFoundException;
+import com.store.pharmacy.utils.BizPartnerConstant;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import java.util.List;
+
+@Service
+@Transactional
+public class BizPartnerServiceImpl implements BizPartnerService {
+
+	@Autowired
+	BizPartnerRepository bizPartnerRepository;
+
+	@Autowired
+	private CommonRepository commonRepository;
+
+	@Autowired
+	private LabelService labelService;
+	private final String INSERT = "INSERT";
+	private final String UPDATE = "UPDATE";
+	private final String SEARCH = "SEARCH";
```

Comment:
Hi @vutch308 ,
Could you please use PharmacyConstant class to call fields: INSERT, UPDATE and add SEARCH field into  PharmacyConstant class?

Reply:
Thanks @NguyenMinhThien97 , I will change this